app/policies/user_policy.rb

```diff
@@ -93,4 +93,10 @@ def check_ownership
   def allow_user_actions?
     user.admin? || user.local_admin? || user.dir_greff_bex? || user.dir_greff_sap?
   end
+
+  def authorized_role?
+    return true if user.admin?
+
+    record.role != 'admin' # Autorise tous les rôles sauf 'admin' pour les non-administrateurs
```

Comment:
!record.admin?

Reply:
Ah yes of course

Reply:
je dirai meme
```
user.admin? || !record.admin?
```

Reply:
comme ca t'a pas besoin de la guard condition